persistence/persistence-deltaspike/src/main/java/uk/gov/justice/services/persistence/EntityManagerProducer.java

```diff
@@ -11,21 +13,23 @@
 /**
  * Producer of {:link EntityManager} for use with JPA (Delta-spike).
  */
+@ApplicationScoped
 public class EntityManagerProducer {
     private static final String UTC = "UTC";
 
     @PersistenceUnit
     EntityManagerFactory entityManagerFactory;
 
     @Produces
+    @RequestScoped
```

Comment:
How does this impact transactions started from MDBS? Does processing a message count aRequests coped?